index.html

```diff
@@ -119,10 +119,10 @@ <h6 class="text-white mr-3 ml-3" style="font-size:13pt"><span style="color: #f50
                   </p>
                   <h1 class="mb-1 mt-5 pb-2 mr-3 ml-3 text-capitalize font-weight-bold"><span style="color: #f50000">Synfig Studio 2D Animation Hackathon</span>
                   </h1>
-                  <h5 class="text-white font-weight-bold"><span style="color: #f50000"> Registration: 24 Oct 2020 – 16 Nov 2020 </span>|</h5>
+                  <h5 class="text-white font-weight-bold"><span style="color: #f50000"> Registration: 2nd November 2020 – 16th Nov 2020 </span></h5>
```

Comment:
Change 2nd November to 02 Nov. Follow the same convention for other dates

Reply:
Okay